cedar-policy/CHANGELOG.md

```diff
@@ -27,6 +27,7 @@ Cedar Language Version: 4.0
 
 ### Added
 
+- Added `get_entity_literals` API (#1149).
```

Comment:
```suggestion
- Added `get_entity_literals` API to get all entity literals occurring in a policy (#1149).
```

Reply:
I prefer keeping changelogs terse and linking to the PR for people who want to look.